packages/prettier-plugin-java/src/printers/classes.js

```diff
@@ -116,27 +116,29 @@ class ClassesPrettierVisitor {
   }
 
   classBody(ctx) {
-    const classBodyDeclsVisited = reject(
-      this.mapVisit(ctx.classBodyDeclaration)
-    );
-    const classBodyDecls = handleClassBodyDeclaration(
-      ctx.classBodyDeclaration,
-      classBodyDeclsVisited
-    );
-
     let content = "";
-    if (classBodyDecls.length !== 0) {
+    if (ctx.classBodyDeclaration !== undefined) {
```

Comment:
``` if (ctx.classBodyDeclaration) {``` ?

Reply:
it is a bit faster to specify it (otherwise, will test if different from "", [], ....), I think. 

As I don't find that it clarifies the code to remove it, I tend to always specify it